logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/call/UpdateVideoStateUseCaseTest.kt

```diff
@@ -20,29 +23,127 @@ class UpdateVideoStateUseCaseTest {
     @Mock
     private val callManager = mock(classOf<CallManager>())
 
+    @Mock
+    private val callRepository = mock(classOf<CallRepository>())
+
     private lateinit var updateVideoStateUseCase: UpdateVideoStateUseCase
 
     @BeforeTest
     fun setup() {
-        updateVideoStateUseCase = UpdateVideoStateUseCase(lazy { callManager })
+        updateVideoStateUseCase = UpdateVideoStateUseCase(lazy { callManager }, callRepository)
+        given(callRepository)
+            .function(callRepository::updateIsCameraOnById)
+            .whenInvokedWith(eq(conversationId.toString()), eq(isCameraOn))
+            .thenDoNothing()
```

Comment:
Couldn't we just use
```
@Mock
val callRepository = configure(mock(classOf<CallRepository>())) { stubsUnitByDefault = true }
```
? 🤔 

Also, probably it would be beneficial if the [Arrangement](https://wearezeta.atlassian.net/wiki/spaces/AR/pages/122520039/Code+Style+Guideline#Unit-Testing-Pattern) which we decided to adopt on last collective was used here

Reply:
I need to assert later that `updateIsCameraOnById` is being invoked with these provided params